src/app/pages/articles/articles.component.html

```diff
@@ -1,17 +1,17 @@
 <h1 class="main-title">Sugar Rush : Votre actu sucrée au quotidien !</h1>
 <div class="top">
-    <a routerLink="{{APP_ROUTES.articles}}/{{ articles[0].slug }}"><app-article-type-main class="main"></app-article-type-main></a>
+    <a routerLink="{{APP_ROUTES.articles}}/{{ articles[0].slug }}"><app-article-type-main [article]="articles[0]" class="main"></app-article-type-main></a>
```

Comment:
instead of using array indexes, could you add an entry inside the mockup datas like "isMainArticle" : boolean ?